src/roswire/common/msg.py

```diff
@@ -29,13 +29,8 @@
 from .decode import is_simple
 from .. import exceptions as exc
 
-R_TYPE = r"[a-zA-Z0-9_/]+(?:\[(?:<=)?\d*\])?"
-R_NAME = r"[a-zA-Z0-9_/]+"
-R_VAL = r".+"
+
 R_COMMENT = r"(#.*)?"
```

Comment:
Does it make sense to move the remaining regexes below?

Reply:
`R_COMMENT` is used in both `Field` and and `MsgFormat`. I think leaving them as global for now is fine. And I'll keep them in the common package for the subclasses too.